template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I'll have some more for you here to hook up ports, and supported auth options based on another PR. Watch this space.